test/XRP/serializer.test.ts

```diff
@@ -923,4 +924,18 @@ describe('serializer', function (): void {
     // WHEN it is serialized THEN the result is undefined.
     assert.isUndefined(Serializer.currencyToJSON(currency))
   })
+
+  it('Serializes a ClearFlag', function (): void {
+    // GIVEN a ClearFlag with no fields set.
```

Comment:
doesn't this have one field set?  the `value` ?

Reply:
Done